frontend/src/scenes/saved-insights/SavedInsights.tsx

```diff
@@ -671,6 +683,13 @@ export function SavedInsights(): JSX.Element {
                         <SavedInsightsEmptyState />
                     ) : (
                         <>
+                            {draftQuery?.query && (
+                                <div className="text-muted-alt mb-4">
```

Comment:
Might make sense to show this on the new insight page as well?
when someone tries to create a new insight again (clicking the + button without going to the saved insights page) they might want to go back to the previous state they had